src/main/java/uk/gov/cslearning/record/repository/InviteRepository.java

```diff
@@ -10,6 +10,6 @@
 @Repository
 public interface InviteRepository extends CrudRepository<Invite, Integer> {
 
-    @Query("SELECT i FROM Invite i INNER JOIN Event e ON i.event = e WHERE e.eventUid = ?1")
+    @Query("SELECT i FROM Invite i INNER JOIN Event e ON i.event = e WHERE e.uid = ?1")
```

Comment:
INFO - is JPA not smart enough to resolve this without explicit DSL?

Reply:
Absolutely See comment [here](https://github.com/Civil-Service-Human-Resources/lpg-learner-record/pull/50#discussion_r233080512). 

I can make the changes if you like but it would probably involve rewriting some of the invitee work.

Reply:
Please take a note, we will have to visit the invitee work in another branch very quickly. Leave as is.